oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/Column.java

```diff
@@ -55,5 +55,12 @@
     /**
      * The column is just saved, never used in query.
      */
-    boolean content() default false;
+    boolean storageOnly() default false;
```

Comment:
I suggest the storageOnly default is `true` in the Column annotation? Because I think most of the columns are just for storage.

Reply:
No, you can't. There are countless place in the codes, including the OAL generated codes. If you use this, we are facing more changes.